components/checkbox/src/index.js

```diff
@@ -88,23 +88,25 @@ const StyledLabel = styled('span')({
 });
 
 const Checkbox = ({
-  children, className, inline, ...input
+  children, inline, disabled, defaultChecked, ...props
 }) => (
-  <StyledCheckbox className={className} inline={inline}>
-    <StyledInput type="checkbox" {...input} />
+  <StyledCheckbox inline={inline} {...props}>
+    <StyledInput type="checkbox" disabled={disabled} defaultChecked={defaultChecked} />
```

Comment:
this is where i think we should have `{...props}`

which means that you wouldn't need to explicitly extract `disabled` and `defaultChecked`